content/en/docs/getting-started.md

```diff
@@ -36,6 +36,21 @@ $ rebar3 local upgrade
 ===> Add to $PATH for use: export PATH=$PATH:~/.cache/rebar3/bin 
 ```
 
+### Windows
+
+Windows users who want to use the code from PowerShell or cmd.exe (rather than a terminal emulator) must ensure that a `rebar3.cmd` file is added:
+
+   ```
+   @echo off
+   setlocal
+   set rebarscript=%~f0
+   escript.exe "%rebarscript:.cmd=%" %*
+   ```
+
+Then add the directory where the file has been saved to the system `PATH`.
```

Comment:
Do the `rebar3` file and the `rebar3.cmd` script go in the same directory?

Reply:
yeah by default this is where it goes. The .cmd file is used to work around the fact that escript expects a `#!/...` in the first line to work, which Windows won't understand. The workaround with rebar3.cmd is that powershell will be able to automatically insert the '.cmd' after 'rebar3', and run it in a way that reinvokes 'escript.exe <rebar3>' and runs properly.

Arguably it could go anywhere you want as long as both are in your PATH variable. I just tend to keep them both at the same place.

Reply:
Thank you

Reply:
I plan to make github actions upload a windows script, should it be `rebar3.cmd`? Confused by whats discussed here. Are there two different files needed dependingon if its powershell or bat?

Reply:
Powershell is gonna work fine with the .cmd script, and it's super small so for the website or anything it should be fine. The powershell script itself is a bit more modern but not strictly necessary. It's just that it's much nicer to use as a language than batch files so it should be what we prefer maintaining in the future.